modules/@apostrophecms/rich-text-widget/index.js

```diff
@@ -324,12 +250,88 @@ module.exports = {
       },
       // Add on the core default options to use, if needed.
       getBrowserData(_super, req) {
-        const result = _super(req);
+        const initialData = _super(req);
 
-        _.defaults(result, {
+        const rteData = {
+          components: {
```

Comment:
If we go with the `${name}Component` pattern we can set the original defaults as options which is more self-documenting and can potentially be extracted for documentation.

Reply:
[See above](https://github.com/apostrophecms/apostrophe/pull/2766/files?file-filters%5B%5D=.js#r584841341). I don't think we should make this call in this PR. I can return these to the options object in a `components` option.